src/internal/task/task.proto

```diff
@@ -30,3 +30,9 @@ message Claim {}
 message TestTask {
   string id = 1 [(gogoproto.customname) = "ID"];
 }
+
+message TaskKey {
```

Comment:
I'm starting to think the key should just be an implementation detail, and we might be better served by just reporting the namespace / group. If that seems reasonable, then we can have a group structure similar to the taskapi one or just return the namespace and group in the callback. Either would be fine for the time being.

Reply:
I was thinking it would be good to have some way of getting to the underlying task just in case, but I can't really think of a case where we'd want that.